tools/version-diff-tool/main.go

```diff
@@ -0,0 +1,297 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/hashicorp/go-version"
+	"github.com/tidwall/sjson"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+const (
+	// File names
+	outputFileName          = `diff_%s_%s.txt`
+	descriptionlessFileName = `r%s`
+	bundleFileName          = `bundle-%s.yaml`
+	openAPIFileName         = `open-api-%s`
+
+	// Splitters
+	schemaSplitter = `---`
+
+	// Links
+	bundleDownloadLink = `https://repository.hazelcast.com/operator/` + bundleFileName
+
+	// Helm related
+	helmInstall      = `helm install operator hazelcast/hazelcast-platform-operator --dry-run --set installCRDs=true > ` + bundleFileName
```

Comment:
```suggestion
	helmInstall      = `helm template operator hazelcast/hazelcast-platform-operator-crds --version=%s > ` + bundleFileName
```

I think we don't need to call the server for rendering so we could use the `template` command instead of `install`.
Also, one potential problem is not setting the chart version; when we have new releases, our helm commands for the operator chart will use the latest version by default. So we need to pass the version explicitly.